### ISSUE TITLE
Keyboard user cannot access overflow menu items in Data Table v2 component

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1470]
## Environment
Chrome 70
macOS Mojave

## Violation
The user can fire the overflow menu on a Data Table row, but cannot enter the menu or select the menu items inside.

**Comment by asudoh** [comment]
Need to update the markup, putting `data-flowing-menu-primary-focus` to the `<button>` of first item.

**Comment by aparnacyanam** [comment]
@asudoh Is there any expected deadline to fix this issue?

**Comment by jnm2377** [comment]
@dakahn @aparnacyanam Is this issue about how the overflow menu doesn't bring the first item to focus and it should? Or just that the menu items aren't accessible? I'm looking at the latest vanilla version locally and I'm able to access the items by using my arrow keys, and selecting by hitting enter. Is seems like that's what this issue is about but I'm not sure if I'm missing something.

![table-overflow](https://user-images.githubusercontent.com/32556167/52963152-a8029b00-3364-11e9-9b60-40c8ffb693f2.gif)

**Comment by aparnacyanam** [comment]
@jnm2377 With keyboard overflow menu is accessible(what you have shown in the above comment is completely working) but click on those functions is not working. In our product, we have features like redirecting to a new page or opening a modal or calling a click function, when you click on the overflow menu option which is not keyboard accessible. This particular issue has blocked our product from passing the accessibility test and hence needed a fix at the earliest possible. Let me know if you need an example, I can forward you our product working env link.

**Comment by jnm2377** [comment]
For reviewers reference: the issue is that overflow menu items are not triggering something (i.e. modal opens, or link takes you somewhere) on keydown, but they do on click. Seems to be related to the keydown event listener.

Here's an example: https://codepen.io/anon/pen/pGGbyR

**Comment by jnm2377** [comment]
Also, this may be an overall overflow menu issue not just data table, since data table is just using overflow menu markup/js.
